### PR TITLE
Wsu standard with retire

### PR DESCRIPTION
**Pull request by mmascher**

### CODE REVIEW

**Comment by mmascher** [comment]
Spawned from https://github.com/opensciencegrid/topology/pull/935

**Comment by mmascher** [comment]
I just realized I need new IDs for the new WSU resource. I'll update the PR after dinner.

File: topology/Wayne State University/WSU - GRID_ce2/GRID_ce2.yaml [hunk 1]
```diff
@@ -0,0 +1,51 @@
+Disable: false
+Production: true
+SupportCenter: Self Supported
+GroupID: 472
```
Comment (brianhlin) [inline topology/Wayne State University/WSU - GRID_ce2/GRID_ce2.yaml]:
```suggestion
GroupID: 521
```

File: topology/Wayne State University/WSU - GRID_ce2/GRID_ce2.yaml [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+GroupDescription: The WSU-GRID_ce2 cluster
+Resources:
+  OSG_US_WSU_GRID_ce2:
+    ID: 895
```
Comment (brianhlin) [inline topology/Wayne State University/WSU - GRID_ce2/GRID_ce2.yaml]:
```suggestion
    ID: 1036
```

File: topology/Wayne State University/WSU - GRID_ce2/GRID_ce2.yaml [hunk 3]
```diff
@@ -0,0 +1,51 @@
...
+      Security Contact:
+        Primary:
+          ID:  f84214412f7b3dbf848808c4ba30d179c5743766
+          Name: Michael Thompson
+    Description: squid service for frontier and cvmfs
+    FQDN: vsquid1.nas.grid.wayne.edu
+    ID: 1026
+    Services:
+      Squid:
+        Description: Generic squid service
```
Comment (brianhlin) [inline topology/Wayne State University/WSU - GRID_ce2/GRID_ce2.yaml]: This looks like a copy-paste error?

File: topology/Wayne State University/WSU - GRID_ce2/SITE.yaml [hunk 4]
```diff
@@ -0,0 +1,13 @@
+LongName: Wayne State University
+Description: The Wayne State University WSU-GRID_ce2 cluster
+
+ID: 10229
```
Comment (brianhlin) [inline topology/Wayne State University/WSU - GRID_ce2/SITE.yaml]:
```suggestion
ID: 10261
```

File: topology/Wayne State University/WSU - GRID_ce2/GRID_ce2.yaml (hunk 3 above)
Comment (mmascher) [reply topology/Wayne State University/WSU - GRID_ce2/GRID_ce2.yaml]: I was not sure if it needed to be moved to the new resource. Removing it from here for the time being.

**Comment by mmascher** [comment]
Suggested changes applied @brianhlin

**Pull request merged**